docs/routes/getting-started/what-is-solidstart.md

```diff
@@ -8,8 +8,7 @@ active: true
 # What is SolidStart?
 
 <aside title="Beware of dragons" type="warning">
-  We are very excited about SolidStart, but it is currently in release candidate. There will very likely be <b>bugs and
-  missing documentation</b>. Come at it with a growth attitude, have fun, and don't use it yet for
+  We are very excited about SolidStart, but it is currently in release candidate. There will very likely be <b>bugs and missing documentation</b>. Come at it with a growth attitude, have fun, and don't use it yet for
```

Comment:
Strange wording "in release candidate", maybe better use "a release candidate".

Reply:
Good point. I added "stage" - does that work?